trace/noop/noop_test.go

```diff
@@ -101,6 +101,22 @@ func TestTracerStartPropagatesSpanContext(t *testing.T) {
 	assert.False(t, span.IsRecording(), "recording span returned")
 }
 
+func BenchmarkNoopInstance(b *testing.B) {
+	tracer := NewTracerProvider().Tracer("")
+	ctx := trace.ContextWithSpanContext(context.Background(), trace.SpanContext{})
+
+	b.ReportAllocs()
+	b.ResetTimer()
+
+	for i := 0; i < b.N; i++ {
+		for j := 0; j < 10; j++ {
```

Comment:
Why do we have a second loop?

Reply:
I was just trying to simulate span being passed in context, but it doesn't need to be. Already removed the.